projects/ion/src/lib/chip/chip.component.html

```diff
@@ -8,4 +8,9 @@
   [class]="'chip-' + size"
 >
   {{ label }}
+  <ion-icon
+    color="#06439D"
+    [type]="icon"
+    [size]="size === 'sm' ? 16 : 20"
+  ></ion-icon>
```

Comment:
I think icon should be optional, right?
Also, icon shouldn't be rendered with a fixed color, since when chip is disabled, it should change its color to gray.